pkg/lib/v0_2_0/keptn.go

```diff
@@ -83,3 +87,106 @@ func (k *Keptn) GetShipyard() (*Shipyard, error) {
 	}
 	return &shipyard, nil
 }
+
+// SendCloudEvent sends a cloudevent to the event broker
+func (k *Keptn) SendCloudEvent(event cloudevents.Event) error {
+	if k.UseLocalFileSystem {
+		log.Println(fmt.Printf("%v", string(event.Data())))
+		return nil
+	}
+
+	return k.EventSender.SendEvent(event)
+}
+
+// SendTaskStartedEvent sends a .started event for the incoming .triggered event the KeptnHandler was initialized with.
+// It returns the ID of the sent CloudEvent or an error
+func (k *Keptn) SendTaskStartedEvent(data keptn.EventProperties, source string) (string, error) {
+
```

Comment:
should we introduce a check here whether `k.CloudEvent != nil` to avoid wrong usage of this function? 

Same for `SendTaskStatusChangedEvent` and `SendTaskFinishedEvent`

Reply:
makes sense, I will add a check for that